db/migrate/20190806102256_add_applications_accepted_from_to_courses.rb

```diff
@@ -0,0 +1,15 @@
+class AddApplicationsAcceptedFromToCourses < ActiveRecord::Migration[5.2]
+  def change
+    add_column :course, :applications_accepted_from, :date
+
+    say_with_time 'Setting applications_accepted_from to one held within sites' do
+      Course.all.each do |course|
+        if course.recruitment_cycle.current?
+          course.update(applications_accepted_from: course.site_statuses.first.applications_accepted_from)
```

Comment:
We don't want the first status’s open date, we want the earliest in the set of statuses.